### DISCUSSION TITLE
How to obtain a GGUF file from the BIN file for Whisper.cpp models?

### DISCUSSION THREAD

**Discussion by MislavJuric** [ggml-org/whisper.cpp#2578] [Q&A] [answered]
Hello Whisper community,

I was wondering **how to obtain a GGUF file from the BIN file for Whisper.cpp models**.

Let's say I download a model from [here](https://huggingface.co/ggerganov/whisper.cpp/tree/main). All of these models have the **.bin** file extension. What I want is something like [this](https://huggingface.co/vonjack/whisper-large-v3-gguf/tree/main), where all the models have the **.gguf** extension.

How can I convert the BIN file to the GGUF file?

**Comment by ggerganov** [comment, marked as answer]
`whisper.cpp` does not support the GGUF file format yet. These models in the link are not really in GGUF format.

**Reply by MislavJuric** [reply]
Thank you for responding!

**Reply by bygreencn** [reply]
https://github.com/handy-computer/transcribe.cpp.git support gguf. Does whisper.cpp have a plan to support it?
